implementations/tests/tests/test_conditions.py

```diff
@@ -271,6 +272,20 @@ class correctly identifies merge commits.
         commit = CommitGit(self.items, conds=[MergeExclude()])
         self.assertEqual(merge_exclude.included, commit.conds[0].included)
 
+    def test_set_commits_commit_by_tag(self):
+        """
+        Test whether the set_commits method of the CommitByTag
+        class correctly identifies the commits contain the tag
+        into their message.
+        """
```

Comment:
I think this should be "... correctly identifies the commits whose messages start with the [api] tag"